signingscript/docker.d/passwords.yml

```diff
@@ -4,9 +4,9 @@ $let:
       'COT_PRODUCT == "firefox"': 'project:releng:signing:'
       'COT_PRODUCT == "thunderbird"': 'project:comm:thunderbird:releng:signing:'
       'COT_PRODUCT == "mobile"': 'project:mobile:focus:releng:signing:'
-      'COT_PRODUCT == "application-services"': 'project:mozilla:application-services:releng:signing'
-      'COT_PRODUCT == "app-services"': 'project:mozilla:app-services:releng:signing'
-      'COT_PRODUCT == "glean"': 'project:mozilla:glean:releng:signing'
+      'COT_PRODUCT == "application-services"': 'project:mozilla:application-services:releng:signing:'
```

Comment:
This was working before as the full scope prefix is being specified instead of using this variable.